15_Ansible_test/hosts

```diff
@@ -0,0 +1,7 @@
+[remote]
+scylla ansible_host=192.168.0.120 user=root ansible_ssh_private_key_file=~/.ssh/ansible
+charybdis ansible_host=192.168.0.121 user=root ansible_ssh_private_key_file=~/.ssh/ansible
+
+[remote_admin]
+scylla ansible_host=192.168.0.120 user=admin ansible_ssh_private_key_file=~/.ssh/ansible_admin
+charybdis ansible_host=192.168.0.121 user=admin ansible_ssh_private_key_file=~/.ssh/ansible_admin
```

Comment:
This is a duplicate file with this one  15_Ansible_test/ansible_roles_test/hosts ?
Please remove if this is a duplicate, if not - why it's placed here, and what for you are using 2 inventory files?

Reply:
This is old files from a first Ansible Homework without roles.
All files for a Homework with roles located in DevOps_BigData/15_Ansible_test/ansible_roles_test/